### DISCUSSION TITLE
What's the best way to change presentation font in the Default theme?

### DISCUSSION THREAD

**Discussion by vedranmiletic** [marp-team/marp#96] [Q&A] [answered]
I have used this code previously:

```html
<style>
section {
 font-family: 'Roboto', 'Segoe UI', 'Liberation Sans', 'Helvetica', 'Arial', sans-serif;
}
</style>
```

However, this code doesn't seem to work in marp-cli 1.0 with marp-core 2.0. The CSS above gets included in `<style></style>`, but not as the last item so it gets overridden by the defaults. Any suggestions?

**Comment by yhatt** [comment]
Now Marp team recognizes that as a regression of Marpit framework: https://github.com/marp-team/marpit/issues/288

---

<details>
<summary>Workaround</summary>

#### `:root` psuedo-class selector

As a workaround, you can use [`:root` pseudo-class selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) instead of `section`. `:root` has higher CSS specificity than the original element so you can overload styles more certainly.

> it may be useful to know that :root actually has a higher specificity. Pseudo-class selectors (but not pseudo-elements) have a specificity equal to that of a class, which is higher than a basic element selector.
>
> _-- https://css-tricks.com/almanac/selectors/r/root/#points-of-interest_

```html
<style>
:root {
  font-family: 'Roboto', 'Segoe UI', 'Liberation Sans', 'Helvetica', 'Arial', sans-serif;
}
</style>
```

In general `:root` represents `html` selector, but [Marp recognizes as `section` selector](https://marpit.marp.app/theme-css?id=root-pseudo-class-selector).

#### `!important`

Or you can use simply [`!important`](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception).

```html
<style>
section {
  font-family: 'Roboto', 'Segoe UI', 'Liberation Sans', 'Helvetica', 'Arial', sans-serif !important;
}
</style>
```

</details>

**Comment by yhatt** [comment, marked as answer]
This regression was fixed in the latest patch release Marp Core v2.0.1 and Marp CLI v1.0.1.

**Reply by vedranmiletic** [reply]
Thank you, that was quick!

**Comment by davekch** [comment]
this is the first result when googling how to change the font in marp so here's another snippet for if you want to change the monospace font:

```html
<style>
  section pre, section code {
   font-family: 'FiraCode Nerd Font', monospace;
  }
</style>
```
